### PR TITLE
Implemented Modal for Add Task

### PR DESCRIPTION
**Pull request by jessesmith-13**

### CODE REVIEW

File: client/components/Tasks/AddTasks.jsx [hunk 1]
```diff
@@ -1,35 +1,165 @@
...
+  const [status, setStatus] = useState('');
+  const [company, setCompany] = useState('');
+  const [category, setCategory] = useState('');
 
   const handleChange = ({target}) => {
-    const { value } = target;
-    setChangeText(value);
-    // console.log('test fired')
+    const { value, name } = target;
+    console.log(value);
```
Comment (greta-schock) [inline client/components/Tasks/AddTasks.jsx:30]: rogue console.log

File: client/components/Tasks/AddTasks.jsx [hunk 2]
```diff
@@ -1,35 +1,165 @@
...
-      <input data-testid="input" value={changeText} onChange={handleChange}>
-      </input>
-      <p data-testid="displayText">
-        {changeText}
-        {text}
-      </p>
-    </div>
-  )
+    <>
+      <Button variant="primary" onClick={handleShow}>
```
Comment (greta-schock) [inline client/components/Tasks/AddTasks.jsx:49]: don't forget accessibility

File: client/components/Tasks/AddTasks.jsx [hunk 3]
```diff
@@ -1,35 +1,165 @@
...
+//       <p data-testid="displayText">
+//         {changeText}
+//         {text}
+//       </p>
+//     </div>
+//   )
+// }
+
+// export default Calendar
+
```
Comment (greta-schock) [inline client/components/Tasks/AddTasks.jsx:165]: Do you want to hold on to this commented-out code for reference later?

**Review by mingwencode — APPROVED** [review]
lgtm

**Pull request merged**